cli/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "xstate-cli",
+  "version": "0.0.1",
+  "bin": {
+    "xstate": "./bin/index.js"
```

Comment:
IMHO reusing "xstate" as a name for this can be slightly confusing, perhaps `xstate-cli` would avoid this problem (especially that it would match the package name)? 
```suggestion
    "xstate": "./bin/index.js"
```

Reply:
Or `stately`?

Reply:
it depends what other responsibilities will be added here but for now the VScode extension is under th XState umbrella and not under Stately, so I would say that this CLI should match it for now

Reply:
Might be one for @davidkpiano, since it's technically branding

Reply:
If it is under `xstate`, we should be maintaining it in the `statelyai/xstate` monorepo, under `packages/cli` 🤔

Reply:
Let's discuss in Discord